api/tools/src/main/java/org/pmiops/workbench/tools/ManageClusters.java

```diff
@@ -91,6 +101,42 @@ private static void listClusters(String apiUrl) throws IOException, ApiException
     System.out.println(String.format("listed %d clusters", count.get()));
   }
 
+  private static void describeCluster(
+      String apiUrl, String workbenchProjectId, String workbenchServiceAccount, String clusterId)
+      throws IOException, ApiException {
+    String[] parts = clusterId.split("/");
+    if (parts.length != 2) {
+      System.err.println(
+          String.format(
+              "given cluster ID '%s' is invalid, wanted format 'project/clusterName'", clusterId));
+      return;
+    }
+
+    // Leo's getCluster API swagger tends to be outdated; issue a raw getCluster request to ensure
+    // we get all available information for debugging.
+    ClusterApi client = newApiClient(apiUrl);
+    com.squareup.okhttp.Call call = client.getClusterCall(parts[0], parts[1], null, null);
+    ApiResponse<Object> resp = client.getApiClient().execute(call, Object.class);
+
+    // Parse the response as well so we can log specific structured fields.
+    Cluster cluster = PRETTY_GSON.fromJson(PRETTY_GSON.toJson(resp.getData()), Cluster.class);
+
+    System.out.println(PRETTY_GSON.toJson(resp.getData()));
```

Comment:
nit: why not allow the caller to specify an output stream?

Reply:
Why? Use of this function is pretty tightly scoped to just this CLI in this file, probably indefinitely - I don't see an advantage to generalizing this.